PR_HW-09_Aleksander

AleksanderQA opened the pull request:
Api automated unit tests
Test Case according to Delete method
Added dependencies: Rest Assured, Params, API, maven plugin

AleksanderQA committed:
Created: Commit-HW-09
Api automated unit tests
Test Case according to Delete method
Added dependencies: Rest Assured, Params, API, maven plugin

vladimir-tl commented on src/test/java/Api/RestApiMocked.java:
```diff
@@ -0,0 +1,111 @@
+        get("test-orders/" + orderId)
+                .then()
+                .log().all()
+                .statusCode(200);
```
please use HttpStatus fields

vladimir-tl commented on src/test/java/Api/RestApiMocked.java:
```diff
@@ -0,0 +1,111 @@
+    }
+    @DisplayName("Deletion of invalid order ID")
+    @Test
+    public void deleteInvalidOrderId(){
```
please extend the test name by adding reference to response code

AleksanderQA committed:
Commit HW-10.
Added:
Parametrized API unit test
Extended Test case
Annotations: @CsvSource
@Tag used for "HW-10"
----------------------
Commit HW-09
Changed HttpStatus fields and added clarifications
for method names
